### PR TITLE
Replace rich.console to btlogging.loggin

### PR DESCRIPTION
**Pull request by basfroman**
In module `bittensor/utils/registration.py` I left the console, because a lot of data output is used. This will create a lot of logging.

### CODE REVIEW

File: bittensor/core/extrinsics/registration.py [hunk 1]
```diff
@@ -218,73 +215,71 @@ def register_extrinsic(
                 netuid=netuid, hotkey_ss58=wallet.hotkey.ss58_address
             )
             if is_registered:
-                bt_console.print(
-                    f":white_heavy_check_mark: [green]Already registered on netuid:{netuid}[/green]"
+                logging.info(
+                    f":white_heavy_check_mark: <green>Already registered on netuid:</blue> <blue>{netuid}</blue>"
```
Comment (ibraheem-abe) [inline bittensor/core/extrinsics/registration.py]: I think we need to close green here before starting blue

File: bittensor/core/extrinsics/registration.py [hunk 2]
```diff
@@ -218,73 +215,71 @@ def register_extrinsic(
...
+                    wait_for_finalization=wait_for_finalization,
+                )
+                success, err_msg = result
+
+                if not success:
+                    # Look error here
+                    # https://github.com/opentensor/subtensor/blob/development/pallets/subtensor/src/errors.rs
+                    if "HotKeyAlreadyRegisteredInSubNet" in err_msg:
+                        logging.info(
+                            f":white_heavy_check_mark: <green>Already Registered on subnet <blue>{netuid}</blue></green>."
```
Comment (ibraheem-abe) [inline bittensor/core/extrinsics/registration.py]: Do we need to close green before starting blue

**Review by ibraheem-abe — APPROVED** [review]
Few nits but overall lgtm

**Review by thewhaleking — APPROVED** [review]
Is there a reason to use the colours/emojis in logging?

**Comment by basfroman** [comment]
> Is there a reason to use the colours/emojis in logging?

It's beautiful and informative for users

**Pull request merged**